README.md

```diff
@@ -91,6 +96,7 @@ private void showDialog() {
                 .setTitle("Rate this application")
                 .setDescription("Please select some stars and give your feedback")
                 .setStarColor(R.color.starColor)
+                .setNoteDescriptionTextColor(R.color.starColor)
```

Comment:
Shouldn't a different color be used here instead?

Reply:
updated

Reply:
I think the one in the sample app got updated to `noteDescriptionTextColor` but not this one?

Reply:
right :) corrected